<?xml version="1.0"?>
<launch>

  <group ns="platform_launch">
    <!-- 
      If value set to false, the target nodes will be terminated when user logic is started.
    -->
    <param name="sensor_fusion" value="true" />
    <param name="slam_wrapper" value="true" />
    <param name="ctl_only" value="true" />
    <param name="fsm" value="true" />

    <!--
      If you want to start up camera_left and camera_right,
      you need to stop (set false) slam_wrapper.
    -->
    <param name="camera_left" value="false" />
    <param name="camera_right" value="false" />
  </group>

  <node name="user_template" pkg="user_template" type="user_template.py" output="screen">
    <!-- The parameters to be used in the user's program can be set -->
    <param name="custom_parameter_integer" value="1" />
    <param name="custom_parameter_float" value="2.0" />
    <param name="custom_parameter_string" value="custom" />
    <param name="custom_parameter_boolean" value="true" />
  </node>
</launch>
